parameter/DomainConfiguration.cpp

```diff
@@ -397,8 +397,10 @@ void CDomainConfiguration::validate(const CConfigurableElement* pConfigurableEle
 {
     CAreaConfiguration* pAreaConfigurationToValidate = getAreaConfiguration(pConfigurableElement);
 
-    // Delegate
-    pAreaConfigurationToValidate->validate(pMainBlackboard);
+    if (pAreaConfigurationToValidate != NULL) {
```

Comment:
I must say I do not really like this way of fixing this. With your way, instead of segfaulting the code will behave erratically in case of error. Could getAreaConfiguration avoid returning NULL ? Maybe we could call explicitly `abort()` instead of `assert(0)` in getAreaConfiguration ?